accProcess.py

```diff
@@ -203,6 +203,26 @@ def main():
                             metavar='True/False', default=False, type=str2bool,
                             help="""Save intensity distribution
                              (default : %(default)s)""")
+    parser.add_argument('--psd',
+                            metavar='True/False', default=False, type=str2bool,
+                            help="""Calculate power spectral density for 24 hour 
+                                    circadian period
+                             (default : %(default)s)""")
+    parser.add_argument('--fourierFrequency',
+                            metavar='True/False', default=False, type=str2bool,
+                            help="""Calculate dominant frequency of sleep
+                             (default : %(default)s)""")
+    parser.add_argument('--m10l5',
+                            metavar='True/False', default=False, type=str2bool,
+                            help="""Calculate relative amplitude of most and 
+                                    least active acceleration periods
+                             (default : %(default)s)""")
+    parser.add_argument('--fourierWithAcc',
+                            metavar='True/False', default=False, type=str2bool,
+                            help="""True will do the fourier analysis with 
```

Comment:
fourier analysis ... **of circadian rhythms** ... with ...

Reply:
Resolved by 57cb667413c64552ac27a412315c4690ca3f6683